<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Logon</class>
 <widget class="QDialog" name="Logon">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog#Logon
{
	border-image: url(:/new/prefix1/background.jpg);
}
QDialog#Logon QPushButton
{
	border-image: url(:/new/prefix1/background.jpg);
	border-radius:8px;
}
QDialog#Logon QPushButton:hover
{
	
	border-image: url(:/new/prefix1/backgroundp.jpg);
}</string>
  </property>
  <widget class="QPushButton" name="pushButton_register">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>750</y>
     <width>200</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string>注册</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_u">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>115</y>
     <width>150</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Agency FB&quot;;</string>
   </property>
   <property name="text">
    <string>用户名</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_p1">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>245</y>
     <width>150</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Agency FB&quot;;</string>
   </property>
   <property name="text">
    <string>密码</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_username">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>100</y>
     <width>400</width>
     <height>70</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/background.jpg);</string>
   </property>
   <property name="placeholderText">
    <string>请输入用户名</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_password">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>230</y>
     <width>400</width>
     <height>70</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/background.jpg);</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>请输入密码</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_p2">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>375</y>
     <width>150</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Agency FB&quot;;</string>
   </property>
   <property name="text">
    <string>确认密码</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_r_password">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>360</y>
     <width>400</width>
     <height>70</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/background.jpg);</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>请再次输入密码</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_t">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>20</y>
     <width>200</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 11pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>注册新用户</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_return">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>750</y>
     <width>200</width>
     <height>70</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">selection-background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(0, 0, 0, 255), stop:0.05 rgba(14, 8, 73, 255), stop:0.36 rgba(28, 17, 145, 255), stop:0.6 rgba(126, 14, 81, 255), stop:0.75 rgba(234, 11, 11, 255), stop:0.79 rgba(244, 70, 5, 255), stop:0.86 rgba(255, 136, 0, 255), stop:0.935 rgba(239, 236, 55, 255));</string>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_mail">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>505</y>
     <width>150</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Agency FB&quot;;</string>
   </property>
   <property name="text">
    <string>QQ邮箱</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_mail">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>490</y>
     <width>400</width>
     <height>70</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/background.jpg);</string>
   </property>
   <property name="placeholderText">
    <string>请输入邮箱</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_icode">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>630</y>
     <width>150</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Agency FB&quot;;</string>
   </property>
   <property name="text">
    <string>验证码</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_icode">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>620</y>
     <width>250</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/background.jpg);</string>
   </property>
   <property name="placeholderText">
    <string>请输入验证码</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_geticode">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>620</y>
     <width>150</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>获取验证码</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
